projects/storefrontstyles/scss/components/myaccount/order-history/_replenishment-order-history.scss

```diff
@@ -0,0 +1,200 @@
+
+      &:first-child {
+        padding-inline-start: 0;
+      }
+
+      &:last-child {
+        text-align: right;
+      }
+    }
+
+    td {
+      width: 14.2%;
+      padding: 1.625rem 5px;
+
+      @include media-breakpoint-up(md) {
+        text-align: left;
+        &:first-child {
+          padding-inline-start: 0px;
+        }
+
+        &:last-child {
+          text-align: right;
+        }
+      }
+
+      @include media-breakpoint-down(sm) {
+        width: 100%;
+        display: flex;
+        border-width: 0;
+        padding: 0 1.25rem;
+
+        &:first-child {
+          padding-top: 1.25rem;
+        }
+
+        &:last-child {
+          padding-bottom: 1.25rem;
+          padding-top: 7px;
+        }
+      }
+    }
+  }
+
+  .cx-order-history-header {
+    padding: 40px 0 0 0;
+    color: var(--cx-color-text);
+
+    @include media-breakpoint-down(sm) {
+      padding: 40px 20px 0 20px;
+    }
+  }
+
+  .cx-order-history-thead-mobile {
+    @include media-breakpoint-down(sm) {
+      display: none;
+    }
+  }
+
+  .cx-order-history-code {
+    .cx-order-history-value {
+      text-decoration: underline;
+    }
+  }
+
+  .cx-order-history-placed {
+    text-align: center;
+
+    @include media-breakpoint-down(sm) {
+      text-align: left;
+    }
+  }
+
+  .cx-order-history-status {
+    text-align: center;
+
+    @include media-breakpoint-down(sm) {
+      text-align: left;
+    }
+  }
+
+  .cx-order-history-total {
+    text-align: right;
+
+    @include media-breakpoint-down(sm) {
+      text-align: left;
+    }
+  }
+
+  .cx-order-history-label {
+    text-transform: uppercase;
+    color: var(--cx-color-secondary);
+
+    @include media-breakpoint-down(sm) {
+      @include type('8');
+      min-width: 60%;
+    }
+  }
+
+  .cx-order-history-value {
+    color: var(--cx-color-text);
+
+    @include media-breakpoint-down(sm) {
+      @include type('5');
+      font-weight: $font-weight-normal;
+    }
+  }
+
+  .cx-order-history-form-group {
+    padding: 0;
+    margin-bottom: 0;
+
+    @include media-breakpoint-down(sm) {
+      padding: 1.25rem;
+    }
+  }
+
+  .cx-order-history-sort {
+    &.top {
+      display: flex;
+      justify-content: space-between;
+      padding: 1rem 0;
+      margin: 0;
+
+      @include media-breakpoint-down(sm) {
+        flex-direction: column;
+        padding-top: 0;
+      }
+    }
+
+    &.bottom {
+      display: flex;
+      justify-content: space-between;
+      padding: 2rem 0 1rem 0;
+      margin: 0;
+
+      @include media-breakpoint-down(sm) {
+        flex-direction: column;
+        padding-top: 0;
+      }
+    }
+  }
+
+  .cx-order-history-pagination {
+    @include media-breakpoint-down(sm) {
+      margin: 0 auto;
+    }
+  }
+
+  .cx-order-history-no-order {
+    @include type('5');
+    font-weight: $font-weight-normal;
+    min-height: 415px;
+
+    @include media-breakpoint-down(sm) {
+      min-height: 474px;
+      padding-left: 1.25rem;
```

Comment:
we no longer use -left or -right for padding or margin.

https://sap-cx.slack.com/archives/GAC8PULNN/p1600495477009100